src/wifi/model/dmg-ap-wifi-mac.h

```diff
@@ -11,6 +11,7 @@
 #include "amsdu-subframe-header.h"
 #include "dmg-beacon-dca.h"
 #include "dmg-wifi-mac.h"
+#include "dmg-wifi-scheduler.h"
```

Comment:
i think a forward declaration should be enough

Reply:
The forward declaration is for dmgWifiScheduler with respect to DmgApWifiMac